frontend/src/views/setting/safe/ssl/index.vue

```diff
@@ -243,41 +232,31 @@ const onSaveSSL = async (formEl: FormInstance | undefined) => {
     if (!formEl) return;
     formEl.validate(async (valid) => {
         if (!valid) return;
-        let msg = !form.autoRestart
-            ? i18n.global.t('setting.sslChangeHelper1') + '\n\n\n' + 'qwdqwdqwd'
-            : i18n.global.t('setting.sslChangeHelper2');
-        ElMessageBox.confirm(msg, i18n.global.t('setting.panelSSL'), {
-            confirmButtonText: i18n.global.t('commons.button.confirm'),
-            cancelButtonText: i18n.global.t('commons.button.cancel'),
-            type: 'info',
-        }).then(async () => {
-            let itemType = form.sslType;
-            if (form.sslType === 'import') {
-                itemType = form.itemSSLType === 'paste' ? 'import-paste' : 'import-local';
-            }
-            let param = {
-                ssl: 'enable',
-                sslType: itemType,
-                domain: '',
-                sslID: form.sslID,
-                cert: form.cert,
-                key: form.key,
-                autoRestart: form.autoRestart,
-            };
+        let itemType = form.sslType;
+        if (form.sslType === 'import') {
+            itemType = form.itemSSLType === 'paste' ? 'import-paste' : 'import-local';
+        }
+        let param = {
+            ssl: 'enable',
+            sslType: itemType,
+            domain: '',
+            sslID: form.sslID,
+            cert: form.cert,
+            key: form.key,
+        };
+        let href = window.location.href;
+        param.domain = href.split('//')[1].split(':')[0];
+        await updateSSL(param).then(() => {
+            MsgSuccess(i18n.global.t('commons.msg.operationSuccess'));
             let href = window.location.href;
-            param.domain = href.split('//')[1].split(':')[0];
-            await updateSSL(param).then(() => {
-                MsgSuccess(i18n.global.t('commons.msg.operationSuccess'));
-                let href = window.location.href;
-                globalStore.isLogin = false;
-                let address = href.split('://')[1];
-                if (globalStore.entrance) {
-                    address = address.replaceAll('settings/safe', globalStore.entrance);
-                } else {
-                    address = address.replaceAll('settings/safe', 'login');
-                }
-                window.open(`https://${address}`, '_self');
-            });
+            globalStore.isLogin = false;
+            let address = href.split('://')[1];
+            if (globalStore.entrance) {
+                address = address.replaceAll('settings/safe', globalStore.entrance);
+            } else {
+                address = address.replaceAll('settings/safe', 'login');
+            }
+            window.open(`https://${address}`, '_self');
         });
     });
 };
```

Comment:
Based on the provided source code snippet, there's no immediate indication of any significant problems or issues. However, a few areas to consider for improvement include:

1. **Consistent naming conventions**: Ensure consistency in variable names across different files to avoid confusion and make it easier to review future changes.

2. **Documentation improvements**: The docstring format might be slightly inconsistent. Standardize how documentation strings (doc comments) appear for readability.

These are generally minor details that shouldn't significantly impact functionality; however, they do illustrate some best practices of coding style guidelines that could potentially improve maintainability and clarity over time. Keep this mind while making further iterations!